package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "auto-remediate",
   "version": "1.0.0",
+  "engines": {
```

Comment:
This should be at the bottom of the JSON to be consistent with most of the other repos